crates/ide/src/references.rs

```diff
@@ -123,7 +123,7 @@ fn retain_import_usages(usages: &mut UsageSearchResult, sema: &Semantics<'_, Roo
 
             if let Some(node) = maybe_node {
                 let res = match node {
-                    syntax::NodeOrToken::Node(x) => {
+                    syntax::NodeOrToken::Node(x) => !matches!(x.kind(), USE),
```

Comment:
Sorry, I tried to make that suggestion for the whole block, which usually works, but didn't this time. The new code doesn't compile, the idea was to replace everything up to the `Token` arm.

Reply:
No problem. Its because of my force-push. You was too fast ) I'll fix this